{"author":"439205512425504771","bot":true,"channel":"343299242963763200","content":"```py\nTraceback (most recent call last):\n  File \"source_file.py\", line 1, in <module>\n    lsusers\nNameError: name 'lsusers' is not defined\n```","created":1528242433524,"id":"453706392897847306","server":"343179912196128792"}